src/Build/Instance/ProjectItemGroupTaskItemInstance.cs

```diff
@@ -229,6 +258,26 @@ public string Remove
             { return _remove; }
         }
 
+        /// <summary>
+        /// Unevaluated MatchOnMetadata value
+        /// </summary>
+        public string MatchOnMetadata
+        {
+            [DebuggerStepThrough]
+            get
+            { return _matchOnMetadata; }
+        }
+
+        /// <summary>
+        /// Unevaluated MatchOnMetadataOptions value
+        /// </summary>
+        public string MatchOnMetadataOptions
```

Comment:
I think you could make this a little cleaner by getting rid of _matchOnMetadataOptions like:
```suggestion
        public string MatchOnMetadataOptions { [DebuggerStepThrough] get; }
```
Same for others.